src/odemis/driver/actuator.py

```diff
@@ -22,19 +22,22 @@
 
 from __future__ import division
 
-from past.builtins import basestring
 import collections
-from concurrent import futures
 import copy
+import enum
+import itertools
 import logging
 import math
 import numbers
+import threading
+from concurrent import futures
+
 import numpy
-import itertools
+from past.builtins import basestring
+
 from odemis import model, util
 from odemis.model import (CancellableThreadPoolExecutor, CancellableFuture,
-                          isasync, MD_PIXEL_SIZE_COR, MD_ROTATION_COR, MD_POS_COR)
-import threading
+                          isasync, MD_PIXEL_SIZE_COR, MD_ROTATION_COR, MD_POS_COR, roattribute)
```

Comment:
I think you can drop `roattribute` now that executor is not shared with everyone.